project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.16
```

Comment:
Rawls build fails with Scala 1.0. SourceClear is trying to use 1.0 since we don't specify anything else ... therefore, scanning fails. So, I pinned it to 0.13. This is also good for our own developers that might run into a problem.